src/swap/defi/thorchainDa.ts

```diff
@@ -127,9 +122,12 @@ export function makeThorchainDaPlugin(
   const { io, log } = opts
   const { fetch } = io
   // const fetch = nodeFetch ?? io.fetch
```

Comment:
I agree with Matthew that all this leftover `fetch` stuff needs to go. *Especially* since the core just directly exposes `node-fetch` as its `io.fetch` function. If you are saying `io.fetch` doesn't work, then `import 'node-fetch'` shouldn't work any better, because it's the exact same thing. Perhaps the core's version is outdated or something? Whatever the problem is, it should be a one-line fix in the core, not a bunch of leftover junk in the plugins.

Reply:
I went ahead and fixed it in the core, then added a commit to the end of this PR to make the requested changes.